crates/server/src/store/blocklist.rs

```diff
@@ -95,8 +99,45 @@ impl BlocklistAuthority {
             block_message: config.block_message.clone(),
             consult_action: config.consult_action,
             log_clients: config.log_clients,
+            #[cfg(feature = "metrics")]
+            metrics: BlocklistMetrics {
+                entries: gauge!("blocklist_list_entries"),
+                blocked_queries: counter!("blocklist_blocked_queries_total"),
+                logged_queries: counter!("blocklist_logged_queries_total"),
+                total_hits: counter!("blocklist_list_hits_total"),
+                total_queries: counter!("blocklist_queries_total"),
+            },
         };
 
+        #[cfg(feature = "metrics")]
+        {
```

Comment:
If there was a `BlocklistMetrics::new()` it feels like these `describe_*` macro invocations could be moved there too, making it just one `#[cfg(feature = "metrics")]` gating on the type construction, vs one for the construction and one for the describe block.